File: Factory/python factory/Mssql.py
from IDatabase import IDatabase

class Mssql(IDatabase):
    def veri_ekle(self):
        print("Mssql'de veri ekleme gerceklestirildi.")
    def veri_cek(self):
        print("Mssql'de veri cekme gerceklestirildi.") 

File: Factory/csharp factory/Mssql.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace csharp_factory
{
    public class Mssql : IDatabase
    {
        public void VeriCek(){
            Console.WriteLine("Mssql'de veri cekme islemi gerceklestirildi.");
        }
        public void VeriEkle(){
            Console.WriteLine("Mssql'de veri ekleme islemi gerceklestirildi.");
        }        
    }
}